week1_day3/exercise1/Config/Database.go

```diff
@@ -0,0 +1,39 @@
+package Config
```

Comment:
use this https://github.com/speakeasy-api/rest-template-go/ and structure your code accordingly.